src/cmd-supermin-shell

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+prepare_build
+
+RUNVM_SHELL=1 runvm -- bash
```

Comment:
The `bash` doesn't actually do anything, no? E.g.

```suggestion
RUNVM_SHELL=1 runvm
```

?

Reply:
I tried that but it fails because "$@" is unset...do we care enough to rework it?